.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: Build and test
+
+on: push
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
```

Comment:
Not sure -- did you need this?

Reply:
This was suggested by Actions template available on GitHub for Gradle. Probably better to keep it if you don't mind.